### ISSUE TITLE
Add image enhancement tips to Information for Transcribers

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#1260]
In view of recent discussion on the transcribers group, might be worth adding some tips to the Information page

**Comment by SteveBiggs** [comment]
Yes this would be useful. Some transcribers have posted some very useful tips on which image viewers to use and how to get the best out of them. We should collate this and add to the help pages.

**Comment by AlOneill** [comment]
Basic information/help is already given on the "Dealing with Uncertainty" page, https://www.freereg.org.uk/cms/information-for-transcribers/dealing-with-uncertainty#imageViewers

If anything in that section is wrong or unhelpful, let me know so that it can be corrected ASAP.

If we want more detailed information, then I suspect that collation alone may not be enough:  the nature of the issue with a particular image, the transcriber's hardware and the transcriber's eyesight all contribute to determining how best to adjust an image for optimal viewing.

Historians, Social Scientists, Archivists and others all face the same issues as us: we should also consider looking for help that is already out there: YouTube or other internet resources.

**Comment by Sherlock21** [comment]
To be honest, I would not have expected to look under the tile: Dealing with Uncertainty to find info about source image enhancing software.

SO perhaps, if others have the same vies as Me,( and as this request was raised in the first place, then some are lost) I would either  change title to include image viewing soft or similar, OR  add its own section.

**Comment by SteveBiggs** [comment]
I think the information about image enhancement software in that section is about the right level. It tells them about a few common packages and the basic controls that should be useful. EricB makes a good point about the title. Perhaps rather than "Dealing with Uncertainty" the top level title would be better called "Reading Difficult Registers"? The content of the page is fine.

**Comment by Sherlock21** [comment]
I am happy with that idea instead.

**Comment by AlOneill** [comment]
If you hover over any link in the Transcriber Help menu, you can see a summary of what that page covers. For the Dealing with Uncertainty page, the summary is "Viewing and reading registers; recording the un-readable".

There is a more detailed listing of the contents of all the pages at the foot of the "Information for Transcribers" page: the listing is described and linked at the top of the page.

While not averse to changing the name of a page, bear in mind that it is quite a lot of work to track down and update all the links: it has to be done manually, a page at a time. If we are going to change the name, let's get the change right the first time!

I gave the page name a lot of thought when first set up. It was tricky to find a name that did not appear to exclude some of the content. "Reading Difficult Registers" appears to exclude the section on UCF. But perhaps my brain works differently from most!
